src/index.js

```diff
@@ -24,7 +24,7 @@ import "whatwg-fetch";
 const saga = createSagaMiddleware();
 const middlewares = [saga];
 if (!isProd) {
-  middlewares.push(logger);
+  //middlewares.push(logger);
```

Comment:
Is this intentionally commented out?

Reply:
I think this could be useful sometimes but I almost always want it turned off during the development mode because it's very verbose. I can uncommit this change